web/src/views/StationView.vue

```diff
@@ -273,8 +306,50 @@ const availabilities: Ref<{
   }
 }> = ref({})
 
+const expandedYears = ref<Set<number>>(new Set([2025]))
+
+function toggleYear(year: number, e: Event) {
+  if (expandedYears.value.has(year)) {
+    expandedYears.value.delete(year)
+  } else {
+    expandedYears.value.add(year)
+  }
+
+  const el = e.currentTarget as HTMLElement
+
+  setTimeout(() => scrollToDivById(el.id, 100), 200)
+}
+
+const orderedWeekdays = ['Dim', 'Sam', 'Ven', 'Jeu', 'Mer', 'Mar', 'Lun']
+
+// Group labels by year and day of week
+const dateGroups = computed(() => {
+  const result: Record<number, Record<string, Array<string | null>>> = {}
+  for (const label of labels) {
+    if (label === 'current') continue
+    const year = Number(label.slice(0, 4))
+    if (!result[year]) result[year] = {}
+    if (objEmpty(result[year])) {
+      const count = Number(dayjs(label).locale('fr').day())
+      for (let i = 0; i < 7 - count; i++) {
+        const weekday = orderedWeekdays[i].toLowerCase()
+        if (!result[year][weekday]) result[year][weekday] = [null]
+      }
+    }
+
+    const weekday = dayjs(label).locale('fr').format('ddd').slice(0, 3) // "Lun", etc.
```

Comment:
If you have a better solution than this, i would take it 
`format('ddd')` returns `Lun.` so i remove the extra dot at the end

Reply:
bizarre le extra '.'